server/vulnerabilities/nvd/cve_test.go

```diff
@@ -312,6 +312,10 @@ func TestTranslateCPEToCVE(t *testing.T) {
 			},
 			continuesToUpdate: true,
 		},
+		"cpe:2.3:a:apple:safari:17.0:*:*:*:*:macos:*:*": {
+			excludedCVEs:      []string{"CVE-2023-28205"},
+			continuesToUpdate: true,
+		},
```

Comment:
Should we add a test to check that we didn't cause any false negatives with this change? e.g. `"cpe:2.3:a:apple:safari:16.4.0:*:*:*:*:macos:*:*"` should be vulnerable to `CVE-2023-28205`.

Reply:
good call, i missed it had a separate safari rule